src/Blazor.EventGridViewer.ServerApp/Controllers/EventGridController.cs

```diff
@@ -38,6 +39,21 @@ public string Get()
             return "EventGridController is running...";
         }
 
+        [HttpOptions]
```

Comment:
@tpaulshippy , can you provide some type of documentation in the README in the root directory.  Maybe the link you provided in the PR, with a brief explanation?